cmd/kubeapps-apis/server/server.go

```diff
@@ -94,28 +94,17 @@ func Serve(serveOpts core.ServeOptions) error {
 	if err != nil {
 		return fmt.Errorf("failed to initialize plugins server: %v", err)
 	}
-	pluginsGRPCv1alpha1.RegisterPluginsServiceServer(grpcSrv, pluginsServer)
-	err = pluginsGRPCv1alpha1.RegisterPluginsServiceHandlerFromEndpoint(gwArgs.Ctx, gwArgs.Mux, gwArgs.Addr, gwArgs.DialOptions)
-	if err != nil {
-		return fmt.Errorf("failed to register core.plugins handler for gateway: %v", err)
-	}
-
 	// Ask the plugins server for plugins with GRPC servers that fulfil the core
 	// packaging v1alpha1 API, then pass to the constructor below.
 	// The argument for the reflect.TypeOf is based on what grpc-go
 	// does itself at:
 	// https://github.com/grpc/grpc-go/blob/v1.38.0/server.go#L621
```

Comment:
This comment needs to be moved with the line below into the function (it's documenting where the crazy reflect.TypeOf arg comes from)

Reply:
yep. you got it